pycbc/frame/frame.py

```diff
@@ -300,10 +284,8 @@ def frame_paths(frame_type, start_time, end_time, server=None, url_type='file'):
     >>> paths = frame_paths('H1_LDAS_C02_L2', 968995968, 968995968+2048)
     """
     site = frame_type[0]
-    connection = datafind_connection(server)
-    connection.find_times(site, frame_type,
-                          gpsstart=start_time, gpsend=end_time)
-    cache = connection.find_frame_urls(site, frame_type, start_time, end_time,urltype=url_type)
+    cache = find_frame_urls(site, frame_type, start_time, end_time,
+                            urltype=url_type, host=server)
```

Comment:
`server` has been added - was this an error before?

Reply:
Previously server was present in the `connection` object. Now the `connection` is set up within `find_frame_urls` function .... So this is not a change in functionality, it just reduces what was two calls, to one.